whois-api/src/test/java/net/ripe/db/whois/api/fulltextsearch/ElasticFullTextSearchTestIntegration.java

```diff
@@ -122,26 +123,25 @@ public void search_single_result() {
         assertThat(queryResponse.getResults().getNumFound(), is(1L));
         assertThat(queryResponse.getResults(), hasSize(1));
         final SolrDocument solrDocument = queryResponse.getResults().get(0);
-        assertThat(solrDocument.getFirstValue("primary-key"), is("1"));
         assertThat(solrDocument.getFirstValue("object-type"), is("mntner"));
         assertThat(solrDocument.getFirstValue("lookup-key"), is("DEV-MNT"));
         assertThat(solrDocument.getFirstValue("mntner"), is("DEV-MNT"));
     }
 
     @Test
-    public void search_single_result_object_deleted_before_index_updated() {
+    public void search_single_result_even_deleted_from_database() {
         final RpslObject mntner = RpslObject.parse(
                 "mntner: DEV-MNT\n" +
                 "source: RIPE");
         databaseHelper.addObject(mntner);
-         rebuildIndex();
+        rebuildIndex();
         databaseHelper.deleteObject(mntner);
 
         final QueryResponse queryResponse = query("q=DEV-MNT");
 
         assertThat(queryResponse.getStatus(), is(0));
-        assertThat(queryResponse.getResults().getNumFound(), is(0L));
-        assertThat(queryResponse.getResults(), hasSize(0));
+        assertThat(queryResponse.getResults().getNumFound(), is(1L));
```

Comment:
I can't believe we we testing for zero when there was one result! 🤦

Reply:
yes, because previously we were checking the object in database...
so this is why the results was 0...because we were comparing what we had in ES and database and that was an inconsistence between both. 
Now we are using just the ES output so that record will be there